src/services/VerifiableCredentialService.ts

```diff
@@ -66,7 +66,9 @@ export class VerifiableCredentialService {
 	}
 	
 
-	async generateSignedVerifiableCredentialJwt(sessionItem: ISessionItem, nameParts: PersonIdentityNamePart[], bankAccountInfo: BankAccountInfo, getNow: () => number): Promise<string> {
+	async generateSignedVerifiableCredentialJwt(
+		sessionItem: ISessionItem, nameParts: PersonIdentityNamePart[], bankAccountInfo: BankAccountInfo, getNow: () => number,
+	): Promise<{ signedJWT: string; evidenceInfo: VerifiedCredentialEvidence }> {
```

Comment:
Shouldnt it it ',' instead of ';' between the 2 properties in the return type?

Reply:
I think it's `;` for an object type, no https://www.typescriptlang.org/docs/handbook/2/objects.html?

Reply:
ok, nice!